jersey/connector/src/main/java/io/helidon/jersey/connector/HelidonConnector.java

```diff
@@ -210,22 +203,10 @@ public ClientResponse apply(ClientRequest request) {
         Http1ClientRequest httpRequest = mapRequest(request);
 
         if (request.hasEntity()) {
-            // if following redirects we need to buffer entity for WebClient
-            if (httpRequest.followRedirects()) {
```

Comment:
Are you positive this is no longer necessary? Certain redirects will require the entity to be sent again. Maybe this is handled elsewhere.

Reply:
Are you saying that a developer should know ahead of time that a redirect is possible and provide an in-memory entity for a potential POST/PUT redirect?

Reply:
buffering into memory should at least be limited by a certain amount of bytes. This is really dangerous.
Also this should not be a feature of the connector, but rather of the webclient - as it takes care of redirection itself (and we do not have it yet implemented, but still responsibility of webclient).

Reply:
@tomas-langer We can certainly included it in the WebClient and set a size limit. I will create a follow-up issue.